components/cli/vendor/github.com/docker/docker/client/client.go

```diff
@@ -260,8 +261,13 @@ func (cli *Client) NegotiateAPIVersionPing(p types.Ping) {
 		p.APIVersion = "1.24"
 	}
 
-	// if server version is lower than the current cli, downgrade
-	if versions.LessThan(p.APIVersion, cli.ClientVersion()) {
+	// if the client is not initialized with a version, start with the latest supported version
```

Comment:
Looks like this was brought into the `components/engine` with git commit https://github.com/docker/docker-ce/commit/1a311235bab8cd31f8b3a63b03c34097f1d52818